courses/free-courses-en.md

```diff
@@ -612,6 +613,7 @@
 * [PHP Programming Language Tutorial - Full Course](https://www.youtube.com/watch?v=OK_JCtrrv-c) - freeCodeCamp.org (2018)
 * [PHP Tutorial](https://www.tutorialrepublic.com/php-tutorial/) - TutorialRepublic
 * [PHP Tutorials](https://www.youtube.com/playlist?list=PL0eyrZgxdwhwBToawjm9faF1ixePexft-) - Dani Krossing
+* [Learn Top Ten Frameworks In PHP By Building Projects](https://geeksgod.com/udemy-free-course/learn-top-ten-frameworks-in-php-by-building-projects/)
```

Comment:
Please:
- Reorder with the purpose of accomplish with the list alphabetization rule.
- Tracking sites / parameters are not accepted. Right link is https://www.eduonix.com/courses/Web-Development/learn-top-ten-frameworks-in-php-by-building-projects
- Should give platform / creator attributions: `Eduonix Learning Solutions`

```markdown
* [Learn Top Ten Frameworks In PHP By Building Projects](https://www.eduonix.com/courses/Web-Development/learn-top-ten-frameworks-in-php-by-building-projects) - Eduonix Learning Solutions
```